### PR TITLE
[TC2-1233] Agreements to use additional properties

### PR DESCRIPTION
**Pull request by kalmerkaurson**

### CODE REVIEW

File: affiliates/argentina.swagger.json [hunk 1]
```diff
@@ -268,6 +268,12 @@
           }
         },
         "Agreements" : {
+          "type" : "object",
+          "additionalProperties" : {
+            "$ref" : "#/components/schemas/Agreement"
```
Comment (Nuffic) [inline affiliates/argentina.swagger.json:273]: image

I'm afraid additionalProperties don't work the way you think they work

**Review by Nuffic — CHANGES_REQUESTED** [review]

File: affiliates/argentina.swagger.json (hunk 1 above)
Comment (kalmerkaurson) [reply affiliates/argentina.swagger.json:273]: If all the properties should be defined in admin then now it should be fine.

Comment (kalmerkaurson) [reply affiliates/argentina.swagger.json:273]: examples for properties

Comment (mikk150) [reply affiliates/argentina.swagger.json:273]: image

Comment (kalmerkaurson) [reply affiliates/argentina.swagger.json:273]: fixed

Comment (kalmerkaurson) [reply affiliates/argentina.swagger.json:273]: no more required arrays

**Comment by mikk150** [comment]
LGTM

**Pull request merged**